RELEASING.md

```diff
@@ -0,0 +1,159 @@
+- In the run up to a new JDK release or quarterly update the OpenJ9 and OMR `master` is branched to create a release branch named according to the OpenJ9 version for that release, e.g. `v0.17.0-release` (In general each quarterly update will have a new OpenJ9 version included)
+- Before a new release there will typically be two milestone builds which get their own git tag from a commit in that branch e.g. `openj9-0.17.0-m1`
+- This milestone tag is then pulled in by an openjdk "extensions" release branch to build an actual JDK binary.
+- When it comes to a GA release day, the release branch will be built with the GA OpenJDK source from the extensions repository, and then tested. If this looks good a tag to indicate the GA level is created in the openj9 repository e.g. `openj9-0.17.0`. This tag is picked up by the openjdk extensions release branch to build the GA JDK binary. You can see the tags at https://github.com/eclipse/openj9/tags
+
+
+### OpenJDK "extensions" repository for OpenJ9 releases
+
+- The openjdk "extensions" repositories e.g. https://github.com/ibmruntimes/openj9-openjdk-jdk11 follows the same branching process as Eclipse OpenJ9 and OMR. When OpenJ9 creates the release branch for the first milestone, a corresponding release branch (based on the `openj9` branch, not `master`) is created, e.g. `openj9-0.17.0`.
+- The "extensions" release branch is updated to pull in the correct OpenJ9 and OMR milestone "tags" to build with for each milestone build.
+- In the run up to the JDK GA date, the extensions team's OpenJDK auto-merge and acceptance jobs merge any new jdk builds into the openj9-staging and openj9 branches, but NOT the openj9-0.nn.0 release branch.
+- When it comes to the GA date, the auto-merged GA jdk "tag" needs to be merged into the openj9-0.nn.0 release branch by the extensions team.
+- The release branch is also updated to pull in the GA Eclipse OpenJ9 & OMR release tags, and then the GA JDK binary is built.
+
+
+# OpenJDK Quarterly/New Release Process
+
+1. Wait for RedHat/Oracle to push the GA code to mercurial and announce availability:
+   * jdk8u : https://hg.openjdk.java.net/jdk8u/jdk8u/jdk/
+     * Announce: https://mail.openjdk.java.net/pipermail/jdk8u-dev/
+   * jdk11+u:  https://hg.openjdk.java.net/jdk-updates/jdk11u/
+     * Announce: https://mail.openjdk.java.net/pipermail/jdk-updates-dev/
+   * jdkNN: https://hg.openjdk.java.net/jdk/jdkNN/
+     * Announce: https://mail.openjdk.java.net/pipermail/jdk-dev/
+
+
+# Extra OpenJ9 prerequisite steps (skip for a HotSpot release)
+
+1. The extensions release branch (e.g. `openj9-0.17.0`) will exist from doing the milestone builds (OpenJ9 milestone process is covered in a later section)
+2. Ask the extensions team to run their release-specific merge jobs to ensure they are up to date - this is not done by jobs at AdoptOpenJDK
+3. Once merged to master, ask the extensions team to merge to openj9-staging branch
```

Comment:
This is done automatically at the same time as merging into master, it is merged into openj9-staging

Reply:
Removed that step now